virtualfish/auto_activation.fish

```diff
@@ -41,5 +41,12 @@ function __vfsupport_deactivate_remove_flag --on-event virtualenv_did_deactivate
     end
 end
 
+# if `vf connect` is run, de-activate when leaving $PWD
+function __vfsupport_auto_deactivate_after_connect --on-event virtualenv_did_connect
+    if not set -q VF_AUTO_ACTIVATED
+        set -g VF_AUTO_ACTIVATED $PWD
+    end
```

Comment:
```suggestion
    set -q VF_AUTO_ACTIVATED; or set -g VF_AUTO_ACTIVATED $PWD
```

Reply:
I mostly wanted an excuse to use GitHub's new multi-line suggestions 😄but this feels neater.

Reply:
Certainly, why not. Thanks for the suggestion.